package/helm/templates/configMap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-config
+data:
+  MB_HOST: "{{ .Values.config.MB_HOST }}"
+  MB_PORT: "{{ .Values.config.MB_PORT }}"
```

Comment:
As discussed, We don't need to configure MB_HOST and MB_PORT from outside now.

Reply:
Yes Done , Thank you